module3/homework/advanced_calculator/advancedCalculator.cpp

```diff
@@ -0,0 +1,77 @@
+#include "advancedCalculator.hpp"
+
+#include <algorithm>
+#include <cctype>
+#include <cmath>
+#include <functional>
+#include <iterator>
+#include <map>
+
+#include <iostream>
```

Comment:
Why iostream include is in another paragraph?

Reply:
@WBigaj you are a real formatting nazist. There is no need to stick to the rule of having headers sorted and without empty newlines. It's just a matter of convention. You can simply approve changes without trying to find such unimportant things.